atlas/responder/pkg/svc/zserver.go

```diff
@@ -0,0 +1,341 @@
+package svc
+
+import (
+	models "atlas/responder/internal"
+	"atlas/responder/pkg/pb"
+	"context"
+	"crypto/sha1"
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+	"sync"
+	"time"
+
+	dapr "github.com/dapr/dapr/pkg/proto/runtime/v1"
+	"github.com/dapr/go-sdk/service/common"
+	"github.com/golang/protobuf/ptypes/empty"
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+const (
+	// version is the current version of the service
+	version            = "0.0.1"
+	invalidServiceName = "invalid service name"
+	emptyRequest       = "empty Request"
+	success            = "success"
+	errorMsg           = "error"
+	errorMissingResp   = "missing repository response error"
+	hiddenUptimeMsg    = "uptime is hidden, mode = false"
+	pubTopic           = "neworder"
+	subTopic           = "neworder2"
+	route              = "/orders2"
+	portal             = "portal"
+	responder          = "responder"
+	storage            = "storage"
+)
+
+type Responder struct {
+	pb.ResponderServer
+	responses *sync.Map
+	logger    *logrus.Logger
+	responder models.Service
+	client    dapr.DaprClient
+	Sub       *common.Subscription
+}
+
+func getResponder() models.Service {
+	return models.Service{
+		ServiceName:          "responder",
+		ServiceDesc:          "responder service desc",
+		ServiceUptime:        time.Now(),
+		ServiceCountRequests: 0,
+	}
+}
+
+func NewResponder(logger *logrus.Logger, s *sync.Map) (*Responder, error) {
+	sub := &common.Subscription{
+		PubsubName: "messages",
+		Topic:      subTopic,
+		Route:      route,
+	}
+
+	conn, err := grpc.Dial(fmt.Sprintf("0.0.0.0:%s", os.Getenv("DAPR_GRPC_PORT")), grpc.WithInsecure())
```

Comment:
Did you use flags `-subscribe-topic` and `-publish-topic` when generated the code? It seems these params have been missed, and due to that the dapr support must have been implemented manually